lib/bootstrap_form/form_builder.rb

```diff
@@ -463,8 +464,8 @@ def generate_label(id, name, options, custom_label_col, group_layout)
         classes << "mr-sm-2"
       end
 
-      unless options.delete(:skip_required)
-        classes << "required" if required_attribute?(object, name)
+      if options.delete(:required) || (!options.delete(:skip_required) && required_attribute?(object, name))
```

Comment:
I take back my earlier comment. Could you please change these lines to the modifier form (`if` at the end of the line). That may fix all the RuboCop offences (even if I wish we didn't have such long lines).